.github/workflows/release.yml

```
@@ -25,7 +25,7 @@ env:
 
     **Documentation**
 
-    Documentation is available on [PHP.net](https://php.net/set.mongodb).
+    Documentation is available on [PHP.net](https://php.net/mongodb).
```

I'm redirected to `www.php.net`
```suggestion
    Documentation is available on [PHP.net](https://www.php.net/mongodb).
```

---

Viewing the source I do see that `https://www.php.net/mongodb` is the displayed value for `shorturl` :+1: 

The final redirect target of both URLs (with or without `www`) is https://www.php.net/manual/en/book.mongodb.php, but I liked the idea of using the short URL since it's more resilient to renaming.